apollo-router-core/Cargo.toml

```diff
@@ -19,36 +20,45 @@ bytes = "1.1.0"
 derivative = "2.2.0"
 displaydoc = "0.2"
 futures = "0.3.19"
+futures-lite = "1.12.0"
```

Comment:
What's `futures-lite` for? Anything we really need from it?

Reply:
I dont think so, removing it 😅